Decent.Minecraft.Client/Blocks/Coal.cs

```diff
@@ -11,6 +14,9 @@ protected Coal(bool charred) : base(BlockType.Coal)
         protected bool IsCharred { get; }
     }
 
+    /// <summary>
+    /// <a href="http://minecraft.gamepedia.com/Coal_Ore">Gamepedia link</a>.
+    /// </summary>
     public class Charcoal : Coal
```

Comment:
No, you're right. I'm the one who got confused: the coal *object* has a charred state, but not the block. You are right, the charred data and the corresponding serialization code should be removed. Sorry about the misunderstanding.